config/crd/bases/elasticdl.org_elasticdljobs.yaml

```diff
@@ -0,0 +1,3749 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.4.0
+  creationTimestamp: null
+  name: elasticdljobs.elasticdl.org
+spec:
+  group: elasticdl.org
+  names:
+    kind: ElasticDLJob
+    listKind: ElasticDLJobList
+    plural: elasticdljobs
+    singular: elasticdljob
+  scope: Namespaced
+  versions:
+  - additionalPrinterColumns:
+    - jsonPath: .status.conditions[-1:].type
+      name: State
+      type: string
+    - jsonPath: .metadata.creationTimestamp
+      name: Age
+      type: date
+    - jsonPath: .spec.ttlSecondsAfterFinished
+      name: Finished-TTL
+      type: integer
+    - jsonPath: .spec.activeDeadlineSeconds
+      name: Max-Lifetime
+      type: integer
+    name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        description: ElasticDLJob Represents an elasticdl Job instance
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this object represents. Servers may infer this from the endpoint the client submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
```

Comment:
I remove `metadata` field manually. Please refer to https://github.com/istio/operator/pull/571/files